book/intro.md

```diff
@@ -1,6 +1,26 @@
 (intro)=
-# Working with modern (cloud) data formats
+# A tutorial for 'Advancing ecosystem carbon flux research'
 
-## A tutorial for 'Advancing ecosystem carbon flux research'
+This book helps to explore tools and datasets, that are useful for researchers working topics ranging from processes in the carbon cycle to machine learning tools and remote sensing. This book will be tested within the ["Advancing ecosystem carbon flux research"](https://www.lorentzcenter.nl/advancing-ecosystem-carbon-flux-research.html) at the Lorentz Center between 14 - 18 October 2024 and be used to inform the discussion on research priorities for the terrestrial carbon cycle and technical developments for research in the next decade. 
 
-Empty
+The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. A longer overview of the material and sections covered in the workshop can be found [here](https://excited-co2.github.io/workshop_tutorial/main/sections/overview.html)
```

Comment:
The link here can be relative. Should be resolved properly when the book is built